src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -244,7 +244,9 @@ def load_and_exec_node(
             node_runs = run_tracker.collect_node_runs()
             if len(node_runs) != 1:
                 # Should not happen except there is bug in run_tracker or thread control.
-                raise NodeResultCountNotMatch("Expecting eactly one node run.")
+                raise UnexpectedError(
+                    message_format="Expecting only one node result. Please contact support for further assistance."
+                )
```

Comment:
To be informative:  Single node execution failed. Expected one node result, but received {node_result_num}. Please contact support for further assistance.

I add template as well in sheet

Reply:
Good catch.